Sources/Purchasing/Offerings.swift

```diff
@@ -93,4 +96,12 @@ public extension Offerings {
         return description
     }
 
+    /**
+     Retrieves a current offering for a placement identifier, use this to access offerings defined by targeting
+     placements configured in the RevenueCat dashboard, e.g. `offerings.getCurrentOffering(for: "placement_id")`.
+     */
+    @objc(getCurrentOfferingForPlacement:)
+    func getCurrentOffering(for placement: String) -> Offering? {
```

Comment:
I know I suggested this, but I think this should be
```suggestion
    func getCurrentOffering(forPlacement placement: String) -> Offering? {
```

Since the type (`String`) doesn't provide enough context on what the parameter is.

Reply:
sure!